bucket/keepassxc.json

```diff
@@ -17,9 +17,23 @@
     },
     "bin": [
         "KeePassXC.exe",
-        "keepassxc-cli.exe"
+        "keepassxc-cli.exe",
+        "keepassxc-proxy.exe"
     ],
-    "persist": "keepassxc.ini",
+    "installer": {
+        "script": [
```

Comment:
Make it one line. It is short enough for Github PR 

![image](https://user-images.githubusercontent.com/13260377/68754012-3d863f00-0606-11ea-84f7-efe1bfa27f48.png)

Reply:
Done